### ISSUE TITLE
Blog Maintainer Role

### ISSUE THREAD

**Issue by Mycelial1** [bisq-network/proposals#81]
> _This is a Bisq Network proposal. Please familiarize yourself with the [submission and review process](https://docs.bisq.network/proposals.html)._

It was made clear in [this issue](https://github.com/bisq-network/compensation/issues/261) that the Bisq blogging initiative was poorly defined in its goals and mechanisms when it was started sometime in March 2019.

After some discussion between @m52go , @arunasurya  and myself, we now have a better idea of the what, why, and how of the Bisq blog.

### #Why:
Blog topics were originally submitted in the Bisq growth repository and I think it speaks to its still-standing goals; to grow the Bisq network.

Without good communications and outreach, Bisq will be slow to adopt. A consistent blogging effort can bring in traffic through search engines, but can also help newcomers be welcomed to Bisq by having a consistent, educational, and friendly voice.

### #What:
We propose that a new role be implemented, titled “Blog Maintainer,” and that this maintainer be given certain duties (I will describe these responsibilities further below.)

That there be a **fixed budget** for writers per voting cycle. The amount is subject to change based on feedback but **can start at 500 BSQ**.

That there be at least **two blog posts** (or however many the stakeholders prefer) written every voting cycle. Compensation will be divided among the writers from the allotted 500 BSQ and can vary per post.

- If the maintainer makes both posts, and deems them worth 250 each, then they earn 500. Full budget spent.
- If they deem their posts worth 100 each, then they earn 200, and 200 is the total amount spent on the blogging initiative that month.
- If they complete one post worth 200, and someone else completes another post worth 250, then each person receives what they earn, and the total spent on blogging that month is 450.
- If one person does work worth 300 and another does work worth 300, then both posts cannot be merged in the same month, since the total of 600 would be higher than the budget of 500. Both posts can still be merged, if stakeholders want them, but they would have to be merged in separate months. It is up to the blog maintainer to make sure the posts merged in a month respect the limits that stakeholders have set.
- Ideal scenario: Blog Maintainer does no writing, and both posts are written by other contributors. In this case, they could charge something symbolic for their role like 25 BSQ (which is the current convention).

### #How

Refer to [my own blog repo](https://github.com/Mycelial1/Blog) that I would like to model this blog process after.

##Duties-
The Blog Maintainer has the following duties:

- Ensure two blogs (or however many stakeholders agree upon) are written and published every voting cycle.
- Assign writers to topics.
- Triage and label submitted topics.
- To include a monthly report that consists of

1. Number of posts targeted per month
2. Actual posts made that month
3. Count of accepted post topic proposals (bounties)

##Rights-
The Blog Maintainer has write access to the new Blog Maintainer repository.

##Owners-
I am volunteering to pick up this role as Primary after discussing with the unofficial blog maintainers Steve Jain and Aruna Surya. More veteran contributors are welcome to be secondary.

I would like to include the following description into the readme doc:

--------------------------------------------------README-----------------------------------------------------
This repository is a workflow for blog posts that are to be written for the https://bisq.network/blog/

Make sure you have joined the[ bisq slack channel](https://bisq.slack.com/messages/CG1R6J5M1/) as well as the #blogposts section.

For styling blog posts in Github, I will provide a markdown template, but you should make use of [this markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).

There will be a **fixed budget of BSQ** to be distributed to Bisq blog contributors every voting cycle (amount is to be determined).

The blog creation process will be as follows:

1. Anyone can submit their blog topic ideas (with a BSQ amount that they think it's worth) as issues in the blog repository.

2. Anyone can upvote or downvote topics and explain why they should or should not be written.

3. If the piece has more upvotes than downvotes and plenty of feedback, then the creator of the issue or anyone else can **claim the piece by commenting on the issue**. The blog maintainer would then **assign the writer to the blog post** and add the writers’ name to the blog writing publication schedule with an assigned publication date.

4. The blog maintainer will wait until a writer has announced they have completed work on a blog topic, so edits can proceed.

5. Any contributor can edit the posts but must do so as pull requests. This allows for transparency when it comes to editors asking for compensation.

6. The blog maintainer will make the final decision on the completeness of the blog posts before they are merged.

7. Writers, editors, and anyone else involved can submit compensation requests for their work.

8. Repeat steps 1 - 7.

Blog topic proposals must include a number of BSQ requested for writing the topic but the amount can change if the writer feels that they deserve more or less for their work.

--------------------------------------------------End of README-----------------------------------------------------

Initial tasks to be done as Blog Maintainer:

Consolidate blog topics under the new Blog repository and label each.
Create the readme doc which describes the blog writing process.
Coordinate and communicate with all current writers about the new writing process.
Create tutorial video describing the Bisq blog writing process.

**Comment by huey735** [comment]
This is tangential to this post but going forward what is the conversion rate to be expected between BSQ and USD? @ManfredKarrer @ripcurlx @m52go  @arunasurya 

That set aside, I feel like only 2 blogs seems low. But this can be altered in the future so there's not much to worry about.

**Comment by arunasurya** [comment]
By the way, I like your initial tasks. I wanted to add a few more thoughts on the role: first, I think that the blog maintainer should be involved in promoting the posts because the main goal of the project is to grow the user base and attract more people. It is best if the maintainer coordinates with the maintainers of comm. channels and ambassadors to promote the post, and do so for all posts. Second, having objective metrics to evaluate the project may be necessary, so tracking the number of views on the Bisq blog site may be a good idea. This can be included in a monthly blog maintainer report. Third, you may want to mention here or in a separate doc that the writers can file for a compensation after the post has been published. Fourth, and most importantly, you may want to add a sentence that some aspects of the role may change depending on how the project goes and that it is up to the maintainer's discretion, since they will know the project best. Some creative freedom and trust in the maintainer's judgment are important, I think.

**Comment by arunasurya** [comment]
@huey735 , you are raising a very good point. I assume that if the price of BSQ compared to USD changes dramatically, that should be reflected in comp. requests. Perhaps you should open an issue to discuss this further?

**Comment by Mycelial1** [comment]
@huey735 BSQ is currently hovering around 19800 sats which is roughly $1.04 USD. Though, it has been pretty volatile.

**Comment by m52go** [comment]
Thanks @Mycelial1 for writing that up.

I think of this role as a specific and well-defined one: make sure the blog is updated on a consistent basis (proposed at 2), that the _maximum_ 'budget' for doing this is an amount accepted by other stakeholders (proposed at 500 USD equivalent of BSQ, contingent on actual work delivered).

That's it. The process to achieve this shouldn't be any different from that of existing Bisq contributions: make an issue for what you want to do, make a pull request with the proposed contribution, get feedback, and then merge.

We will need to figure out how exactly to make this process more friendly for writers who aren't familiar with GitHub, etc., but I don't think we need to determine those exact processes here (John's made a good start with his video).

**Marketing**

Marketing needs to be considered separately. It needs to be done, but that's not a job for this role. Desktop software maintainers aren't tasked with maximizing software downloads. 

I do think that post effectiveness needs to be considered. As website maintainer I have access to website analytics, so going forward I will make sure blog post analytics are posted every month, so people can see which blog posts do well and which don't. Then the blog maintainer (and others who provide feedback on proposed posts) can be smarter about rejecting posts that haven't done well in the past.

Determining a process for how folks can contribute to marketing and messaging for Bisq is a bigger conversation, and if we're going to keep this role's mandate specific and focused, I think it's irrelevant.

**Role Responsibilities**

Once we have requirements and process documented, people just need to follow it. I don't see a need for any active efforts in coordination and communication with writers. I don't know of any publication that solicits guest posts which does anything more than this on a regular basis.

Basically I don't think the blog maintainer should be doing any big work aside from ensuring a good list of blog proposals and labeling/critiquing submitted blog proposals. Compensation for that work should be minimal, perhaps covered in the typical role compensation amount of 25-50 BSQ.

Reviewing posts would be reviewing pull requests, which is subject to compensation (just like it always is) for anyone who does it. And of course, the blog maintainer would be compensated for any posts he actually writes himself.

**Comment by Mycelial1** [comment]
Thanks for all of the feedback everyone.

I think the feedback needed most would be regarding the **budget of BSQ** for the writers per voting cycle (currently suggesting 500 BSQ) and the **minimum amount of blog posts** every cycle (currently at two blog posts).

@huey735 asked about the exchange rate between BSQ and USD. 

I think 500 BSQ is somewhat fair. Compensation for writers can vary highly in other industries but I think this is a good starting point in terms of compensation. 500 would be a hard cap and might not be entirely distributed every cycle.

I think the role's responsibilities are clear (unless someone feels otherwise).

**The role's responsibilities are as follows**:
1. Ensure at least 2 blogs posts are written per month.
2. Maintaining the blog repository.
3. Reviewing blog posts before publication (any other contributor can do this as well but the blog maintainer SHOULD be doing this).

I will disagree with @m52go about communication with writers being unnecessary. I think it's important to be in communication with ANY contributor, but I feel this will naturally happen anyways within Slack and other channels.

I agree with @arunasurya about being in communication with the admins of the social channels so we can maximize blog post reach. 

Though, I would like to approach promotion more like @m52go 's suggestions.

**Marketing**
I agree that marketing and blog creation need to be kept separate from each other. 

Though, properly constructed blogs can do its own marketing if executed correctly.
If we write a search engine optimized post, then the post will do its own marketing via search engines.

If we write, "How Venezuelans can Buy Bitcoin with Bisq" we can then have it translated into Spanish by Sebastien Tellez (or anyone else,) and share it to a Venezuelan audience (I know a few in Discord) which can help Venezuelans learn about Bisq while potentially increasing user adoption.

If we want to attract certain kinds of devs to Bisq, then we should write content for them. 

Every blog idea issue should include sound reasoning for its creation.

**Comment by arunasurya** [comment]
I think it is great that we have these discussions because this can help us figure out how we should proceed with similar projects in the future:). I understand @m52go's point about making the project more decentralized, but I also think that the maintainer will need to be involved in communicating with the writers, ensuring no mistakes in the final work, and coordinating things in general. In the last blog project, we had to reject two posts and ensure that there were no repeating topics, that there was a balance of topics and there were no serious mistakes or misrepresentations.

Perhaps we can start with a total budget for the Blog project of 1000BSQ, which covers two blog posts minimum plus compensation for the maintainer. Maybe we can start with 200-300BSQ for the role, and it can be adjusted depending on the involvement of the maintainer? In any case, we can make a proposal in this DAO cycle, and if the community accepts it, give it a try for one month, and then assess the work done. This way, we will have more info and more posts. The community has to decide if there is value in the posts in the first place, and if there is, assign a budget that encourages good work and initiative.

**Comment by m52go** [comment]
I think it makes sense to mark this proposal as stale, since it hasn't gotten much feedback in the past week.

We'll try again in another 2-3 voting cycles, perhaps with a stronger case, and better timing (I know blogging is not a priority at the moment).

In the mean time, John and I will try to keep the blog going with informal efforts, along with anyone else who would like to help, with the understanding that compensation will be scarce or nonexistent since stakeholders have not indicated interest in the endeavor. New posts will be published on a best-efforts basis.
